JSON/very_small_example_report.json

```diff
@@ -0,0 +1,75 @@
+{
+	"title": "OSCAP Scan Result",
+	"identity": "jrodak",
+	"profile_name": "xccdf_com.example.www_profile_test_single_rule",
+	"platform": "",
+	"target": "rh-hony",
+	"cpe_platforms": [],
+	"scanner": "OpenSCAP",
+	"scanner_version": "1.3.5",
+	"benchmark_url": "#scap_org.open-scap_comp_test_single_rule.xccdf.xml",
+	"benchmark_id": "xccdf_com.example.www_benchmark_dummy",
+	"benchmark_version": "1.0",
+	"start_time": "2021-11-30T12:27:51+01:00",
+	"end_time": "2021-11-30T12:27:51+01:00",
+	"test_system": "cpe:/a:redhat:openscap:1.3.5",
+	"score": 100.0,
+	"score_max": 100.0,
+	"rules": {
+		"xccdf_com.example.www_rule_test-pass": {
+			"rule_id": "xccdf_com.example.www_rule_test-pass",
+			"title": "This rule always passes",
+			"result": "pass",
+			"multi_check": false,
+			"time": "2021-11-30T12:27:51+01:00",
+			"severity": "Unknown",
+			"identifiers": [],
+			"references": [],
+			"description": "Description",
+			"rationale": "",
+			"warnings": [],
+			"platforms": [],
+			"oval_definition_id": "oval:x:def:1",
+			"messages": [],
+			"remediations": [],
+			"oval_tree": {
```

Comment:
This three could became complex. I suggest to include some complex rules in the tests. For example:
- https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_deny/oval/shared.xml

This rule contains multiple tests and variables.

Reply:
I added an example rule [xccdf_org.ssgproject.content_rule_accounts_passwords_pam_faillock_deny](https://github.com/OpenSCAP/openscap-report/blob/0b329c334df63ec53fcaffc028743e8b63fe7ace/JSON/small_example_with_more_complex_rule.json).